CONTRIBUTORS_QUICK_START.rst

```diff
@@ -1110,14 +1112,15 @@ Setting up Breeze
 
   $ ./breeze --python 3.8 --backend mysql
 
-Note: If you encounter an error like "docker.credentials.errors.InitializationError:
-docker-credential-secretservice not installed or not available in PATH", you may execute following command to fix it.
+.. note::
+   If you encounter an error like "docker.credentials.errors.InitializationError:
+   docker-credential-secretservice not installed or not available in PATH", you may execute the following command to fix it:
```

Comment:
```suggestion
.. note::

   If you encounter an error like "docker.credentials.errors.InitializationError:
   docker-credential-secretservice not installed or not available in PATH", you may execute the following command to fix it:
```

I _think_ an empty line is needed? (Feel free to ignore this if Sphinx renders it correctly for you.)

Reply:
Actually for Contributors quick start we should rely on what GitHub does for rendering. We do not build it as part of our documentation

Reply:
Never mind this then. GitHub renders fine.